src/main/java/de/tum/in/www1/artemis/service/connectors/ContinuousIntegrationService.java

```diff
@@ -237,7 +237,7 @@
             @Override
             public String forProgrammingLanguage(ProgrammingLanguage language) {
                 return switch (language) {
-                    case JAVA, PYTHON, C, HASKELL -> Constants.ASSIGNMENT_CHECKOUT_PATH;
+                    case JAVA, PYTHON, C, HASKELL, KOTLIN, VHDL, ASSEMBLER -> Constants.ASSIGNMENT_CHECKOUT_PATH;
                     default -> throw new IllegalArgumentException("Repository checkout path for assignment repo has not yet been defined for " + language);
```

Comment:
```suggestion
```

Reply:
@krusche Should we change this?

Reply:
Yes, in case we cover all cases, the compiler should check the completeness and we should delete the `default` statement.

Reply:
I've removed the default branch.